source/features/expand-all-collapsed-code.tsx

```diff
@@ -0,0 +1,45 @@
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+
+const expanderSelector = '.js-expand.directional-expander';
+
+// When diff table's childlist changes, select and click expand button to load more lines
+const clickOnAutoexpandObserver = new MutationObserver(mutations => {
+	for (const mutation of mutations) {
+		const btn = select(expanderSelector, mutation.target as HTMLElement);
+		if (btn) {
+			btn.click();
+		}
+	}
+});
+
+function unfold(event: DelegateEvent<MouseEvent>): void {
+	if (!event.altKey) {
+		return;
+	}
+
+	const table = (event.target as Element).closest('.diff-table > tbody');
```

Comment:
```suggestion
	const table = (event.target as Element).closest('.diff-table > tbody')!;
```

This will avoid the `null` type error caused by `Node | null`